src/proto/generation.proto

```diff
@@ -59,6 +59,12 @@ message Artifact {
   uint64 size = 13;               // Size of the artifact in bytes
 }
 
+enum MaskedAreaInit {
```

Comment:
We also need to add a `*_to_string` function to the `generator_server`, it will convert the protobuf Enum to python enum and pass to the `stable-diffusion`.

https://github.com/Stability-AI/generator_server/blob/ea86775b271d186bb47389a47af8c3629e36794f/resolver.py#L22

Reply:
yep I have separate change to generator_server 👍, just wanted to check on placement of the setting here before doing PR there.